keps/sig-storage/1710-selinux-relabeling/README.md

```diff
@@ -588,21 +814,23 @@ _This section must be completed when targeting beta graduation to a release._
   determine the health of the service?**
 
   - [x] Metrics
-    - All `errors_total` metrics below cover real errors when a Pod can't start.
+    - All `*_errors_total` metrics below cover real errors when a Pod can't start.
```

Comment:
(suggestion) Can we explicit the labels for each metric?

Again it might be worth clarifying that all metrics are counters. I'm also not sure how the metrics would be used since they are not part of the rules below.

Reply:
Added labels + mentioned counters